### ISSUE TITLE
pt-BR translation suggestion

### ISSUE THREAD

**Issue by rbaprado** [medyo/android-about-page#28]
Here it goes:

```
<resources>
    <string name="about_page_description" translatable="false" >Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus vehicula ligula tortor, in pellentesque dui congue ac. Vestibulum lacinia urna magna, quis cursus ante iaculis nec. Nunc ut ligula sit amet odio ultrices euismod vel vel purus. Sed lacus mauris, aliquam et placerat id</string>

    <string name="about_contact_us">Fale Conosco</string>
    <string name="about_instagram">Siga-nos no Instagram</string>
    <string name="about_facebook">Curta no Facebook</string>
    <string name="about_play_store">Avalie-nos na Play Store</string>
    <string name="about_youtube">Assista-nos no Youtube</string>
    <string name="about_twitter">Siga-nos no Twitter</string>
    <string name="about_github">Crie um Fork no GitHub</string>
    <string name="about_website">Visite o nosso website</string>

</resources>
```
Disclaimer: Portuguese is my native language

**Comment by skmlcd** [comment]
I can make a pull request for you if you want.

**Comment by rbaprado** [comment]
@skmlcd I would appreciate that, thanks.

**Comment by skmlcd** [comment]
Alright, done: 2b9ca45

**Comment by medyo** [comment]
Merged #90e3a01
